### PR TITLE
fix: pick bt device on first load, select non-default version of default device

### PR DESCRIPTION
**Pull request by eswarclynn**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/hms-video-store/src/device-manager/DeviceManager.ts [hunk 1]
```diff
@@ -205,7 +205,12 @@ export class DeviceManager implements HMSDeviceManager {
     }
     const audioDeviceId = this.store.getConfig()?.settings?.audioInputDeviceId;
     if (!audioDeviceId && localPeer?.audioTrack) {
-      await localPeer.audioTrack.setSettings({ deviceId: this.audioInput[0]?.deviceId }, true);
+      const getInitialDeviceId = () => {
+        const isMobile = !('setSinkId' in HTMLMediaElement.prototype);
```
Comment (raviteja83) [inline packages/hms-video-store/src/device-manager/DeviceManager.ts:209]: this willl be true for firefox and safari desktop I think.

Comment (raviteja83) [reply packages/hms-video-store/src/device-manager/DeviceManager.ts:209]: as discussed, if verified on android, we can test with just isIOS

File: packages/hms-video-store/src/device-manager/DeviceManager.ts [hunk 2]
```diff
@@ -416,6 +421,29 @@ export class DeviceManager implements HMSDeviceManager {
...
+    let wired: InputDeviceInfo | null = null;
+    let earpiece: InputDeviceInfo | null = null;
+
+    for (const device of this.audioInput) {
+      const label = device.label.toLowerCase();
+      if (label.includes('speakerphone')) {
+        speakerPhone = device;
+      } else if (label.includes('wired')) {
+        wired = device;
+      } else if (/airpods|buds|bluetooth/.test(label)) {
```
Comment (raviteja83) [inline packages/hms-video-store/src/device-manager/DeviceManager.ts:437]: add gi at the end of regex. pro can also be one that might be common.

Comment (raviteja83) [reply packages/hms-video-store/src/device-manager/DeviceManager.ts:437]: wireless can also be added.

Comment (eswarclynn) [reply packages/hms-video-store/src/device-manager/DeviceManager.ts:437]: label is already converted to lowercase, so 'i' in the regex won't be required?

Will add wireless but how do you think pro is useful, our MacBook Pros can also be misidentified as Bluetooth devices in this case?

Comment (raviteja83) [reply packages/hms-video-store/src/device-manager/DeviceManager.ts]: yeah pro we can ignore. good point.

**Review by raviteja83 — APPROVED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]

**Pull request merged**